actions/views/rhs.ts

```diff
@@ -113,13 +128,28 @@ export function performSearch(terms: string, isMentionSearch?: boolean) {
         const teamId = getCurrentTeamId(getState());
         const config = getConfig(getState());
         const viewArchivedChannels = config.ExperimentalViewArchivedChannels === 'true';
+        const extensionsFilters = getFilesSearchExtFilter(getState() as GlobalState);
+
+        const extensions = extensionsFilters.length > 0 ? ' ' + extensionsFilters.map((ext) => `ext:${ext}`).join(' ') : '';
+        const termsWithExtensionsFilteres = terms + extensions;
```

Comment:
Would it make sense to write this as:
```suggestion
        const extensions = extensionsFilters?.map((ext) => `ext:${ext}`).join(' ');
        const termsWithExtensionsFilteres = [terms, extensions].filter(x => x?.length > 0).join(' ');
```

Also, I note the typo: `Filteres` should be `Filters`.

Reply:
I think is harder to undrestand, I going to change it a bit to make it more explicit.

Reply:
Changed with something else